plugins/kubernetes-backend/src/service/KubernetesFetcher.test.ts

```diff
@@ -500,13 +544,14 @@ describe('KubernetesFetcher', () => {
         customResources: [],
       });
 
-      const mockCall = clientMock.listClusterCustomObject.mock.calls[0];
+      const mockCall =
+        customObjectsClientMock.listClusterCustomObject.mock.calls[0];
       const actualSelector = mockCall[mockCall.length - 1];
       const expectedSelector = 'backstage.io/kubernetes-id=some-service';
       expect(actualSelector).toBe(expectedSelector);
     });
     it('should use namespace if provided', async () => {
-      clientMock.listNamespacedCustomObject.mockResolvedValueOnce({
+      customObjectsClientMock.listNamespacedCustomObject.mockResolvedValueOnce({
```

Comment:
Did you create a test for the case where we return an empty list, when the namespace doesn't exist? Maybe you've already tested it and I've seen it, but if not, how about testing it?

Reply:
Done https://github.com/backstage/backstage/pull/14714/commits/0419dd2adbdb351af7593ed8ad24962c68e2f0de